gcloud/datastore/connection.py

```diff
@@ -429,6 +429,10 @@ def save_entity(self, dataset_id, key_pb, properties,
            will be replaced by those passed in ``properties``;  properties
            not passed in ``properties`` no longer be set for the entity.
 
+        .. note::
+           When saving an entity to the backend, property value set as
```

Comment:
Change to:
When saving an entity to the backend, a property value set as
an empty list or None cannot be saved and will be ignored.